### ISSUE TITLE
Main navigation accessibility / usability issues

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/ffnc-guide#10]
- [x] Navigation headers are marked up as an anchor with no attributes. These should be handled as flyout menu toggles via WAI-ARIA recommendation.
- [x] Flyout menus are activated on keyboard focus. Users may not want this to happen, especially on the mega menu where they have to tab through 20+ items to get to the next parent item Ideally, these should be activated with hover and user input on a keyboard.
- [x] Mobile menu toggle is a form input. This should ideally be set up as a button with ARIA to improve the understanding and functionality (aria-expanded, for example).
